packages/flutter_test/lib/src/controller.dart

```diff
@@ -527,12 +536,94 @@ abstract class WidgetController {
     int buttons = kPrimaryButton,
     double touchSlopX = kDragSlopDefault,
     double touchSlopY = kDragSlopDefault,
+    Duration duration,
+    double frequency = 60.0,
   }) {
-    assert(kDragSlopDefault > kTouchSlop);
```

Comment:
I think assert the value that's going to be used is more meaningful than assert the global constant.